### ISSUE TITLE
Add lightning strikes to app

### ISSUE THREAD

**Issue by tomahock** [FogosPT/fogosmobile#109]
You can find them here: https://fogos.pt/lightnings

**Comment by Vanethos** [comment]
@tomahock is it possible to add the data model here?

Today, for example, since there are no lightnings the data set is empty: 

```json
{"data": []}
```

**Comment by Vanethos** [comment]
Data example:

```json
{"data": [{"timestamp": "2020-06-30 03:07", "payload": {"latitude": 39.888300000000001, "amplitude": -8.6999999999999993, "longitude": -11.537000000000001}}, {"timestamp": "2020-06-30 12:52", "payload": {"latitude": 42.376100000000001, "amplitude": -18.699999999999999, "longitude": -6.5231000000000003}}, {"timestamp": "2020-06-30 12:56", "payload": {"latitude": 42.441499999999998, "amplitude": -18.5, "longitude": -6.5724999999999998}}, {"timestamp": "2020-06-30 12:58", "payload": {"latitude": 42.473500000000001, "amplitude": -12.1, "longitude": -6.4234}}, {"timestamp": "2020-06-30 13:00", "payload": {"latitude": 42.383200000000002, "amplitude": -20.199999999999999, "longitude": -6.5677000000000003}}, {"timestamp": "2020-06-30 13:18", "payload": {"latitude": 42.343400000000003, "amplitude": -18.5, "longitude": -6.5101000000000004}}, {"timestamp": "2020-06-30 13:21", "payload": {"latitude": 42.427300000000002, "amplitude": -19.800000000000001, "longitude": -6.5068000000000001}}, {"timestamp": "2020-06-30 13:21", "payload": {"latitude": 40.516300000000001, "amplitude": -9.8000000000000007, "longitude": -4.7751000000000001}}, {"timestamp": "2020-06-30 13:30", "payload": {"latitude": 42.309399999999997, "amplitude": -16.100000000000001, "longitude": -6.4469000000000003}}, {"timestamp": "2020-06-30 13:30", "payload": {"latitude": 42.460599999999999, "amplitude": -11.5, "longitude": -6.3651}}, {"timestamp": "2020-06-30 13:32", "payload": {"latitude": 42.423299999999998, "amplitude": -11.300000000000001, "longitude": -6.3224}}, {"timestamp": "2020-06-30 13:33", "payload": {"latitude": 42.419899999999998, "amplitude": -18.0, "longitude": -6.3076999999999996}}, {"timestamp": "2020-06-30 13:33", "payload": {"latitude": 42.417999999999999, "amplitude": -15.1, "longitude": -6.3334999999999999}}, {"timestamp": "2020-06-30 13:35", "payload": {"latitude": 42.423099999999998, "amplitude": -13.4, "longitude": -6.3128000000000002}}, {"timestamp": "2020-06-30 13:37", "payload": {"latitude": 42.351799999999997, "amplitude": -7.5999999999999996, "longitude": -6.5671999999999997}}, {"timestamp": "2020-06-30 13:38", "payload": {"latitude": 42.343400000000003, "amplitude": -12.699999999999999, "longitude": -6.6136999999999997}}, {"timestamp": "2020-06-30 13:39", "payload": {"latitude": 42.468400000000003, "amplitude": -8.4000000000000004, "longitude": -6.2271999999999998}}, {"timestamp": "2020-06-30 13:39", "payload": {"latitude": 42.379800000000003, "amplitude": -16.300000000000001, "longitude": -6.2840999999999996}}, {"timestamp": "2020-06-30 13:40", "payload": {"latitude": 42.524999999999999, "amplitude": -10.9, "longitude": -6.1007999999999996}}, {"timestamp": "2020-06-30 13:40", "payload": {"latitude": 42.432499999999997, "amplitude": -14.300000000000001, "longitude": -6.5095000000000001}}, {"timestamp": "2020-06-30 13:41", "payload": {"latitude": 42.531399999999998, "amplitude": -9.3000000000000007, "longitude": -6.1063000000000001}}, {"timestamp": "2020-06-30 13:43", "payload": {"latitude": 42.397100000000002, "amplitude": -16.600000000000001, "longitude": -6.4458000000000002}}, {"timestamp": "2020-06-30 13:46", "payload": {"latitude": 42.415300000000002, "amplitude": -19.399999999999999, "longitude": -6.218}}, {"timestamp": "2020-06-30 13:47", "payload": {"latitude": 42.415900000000001, "amplitude": -19.600000000000001, "longitude": -6.5800999999999998}}, {"timestamp": "2020-06-30 13:47", "payload": {"latitude": 42.311399999999999, "amplitude": -19.399999999999999, "longitude": -6.3338999999999999}}, {"timestamp": "2020-06-30 13:49", "payload": {"latitude": 42.410800000000002, "amplitude": -17.800000000000001, "longitude": -6.2370999999999999}}, {"timestamp": "2020-06-30 13:49", "payload": {"latitude": 42.495600000000003, "amplitude": -6.2000000000000002, "longitude": -6.2168000000000001}}, {"timestamp": "2020-06-30 13:53", "payload": {"latitude": 42.334600000000002, "amplitude": -15.6, "longitude": -6.5823999999999998}}, {"timestamp": "2020-06-30 13:56", "payload": {"latitude": 42.462299999999999, "amplitude": -24.600000000000001, "longitude": -6.2480000000000002}}, {"timestamp": "2020-06-30 13:56", "payload": {"latitude": 42.976599999999998, "amplitude": -14.9, "longitude": -6.1577000000000002}}, {"timestamp": "2020-06-30 13:56", "payload": {"latitude": 42.337600000000002, "amplitude": -32.700000000000003, "longitude": -6.6407999999999996}}, {"timestamp": "2020-06-30 13:56", "payload": {"latitude": 42.356299999999997, "amplitude": -19.699999999999999, "longitude": -6.2671000000000001}}, {"timestamp": "2020-06-30 13:56", "payload": {"latitude": 42.490000000000002, "amplitude": -9.0, "longitude": -6.2081}}, {"timestamp": "2020-06-30 13:59", "payload": {"latitude": 42.604100000000003, "amplitude": -8.0, "longitude": -6.1177999999999999}}, {"timestamp": "2020-06-30 13:59", "payload": {"latitude": 42.935299999999998, "amplitude": 6.0999999999999996, "longitude": -5.9960000000000004}}, {"timestamp": "2020-06-30 14:01", "payload": {"latitude": 42.840600000000002, "amplitude": 25.600000000000001, "longitude": -5.8674999999999997}}, {"timestamp": "2020-06-30 14:02", "payload": {"latitude": 42.958500000000001, "amplitude": -22.600000000000001, "longitude": -5.9078999999999997}}, {"timestamp": "2020-06-30 14:02", "payload": {"latitude": 41.039000000000001, "amplitude": 18.800000000000001, "longitude": -4.476}}, {"timestamp": "2020-06-30 14:02", "payload": {"latitude": 42.816800000000001, "amplitude": 27.399999999999999, "longitude": -5.9108000000000001}}, {"timestamp": "2020-06-30 14:04", "payload": {"latitude": 40.444299999999998, "amplitude": 62.0, "longitude": -5.3394000000000004}}, {"timestamp": "2020-06-30 14:08", "payload": {"latitude": 42.283499999999997, "amplitude": 24.899999999999999, "longitude": -6.5191999999999997}}]}
```

**Comment by Vanethos** [comment]
Closed via #128
